src/steps/user/converter.ts

```diff
@@ -0,0 +1,26 @@
+import {
+  createIntegrationEntity,
+  Entity,
+} from '@jupiterone/integration-sdk-core';
+
+import { Entities } from '../constants';
+import { User } from '../../types';
+
+export function getUserKey(id: string): string {
+  return `asana_user:${id}`;
+}
+
+export function createUserEntity(data: User): Entity {
+  return createIntegrationEntity({
+    entityData: {
+      source: data,
+      assign: {
+        _key: getUserKey(data.gid),
+        _type: Entities.USER._type,
+        _class: Entities.USER._class,
+        username: data.email,
+        active: true,
```

Comment:
@eXtremeX There was no `active` property in the most recent [endpoint](https://developers.asana.com/docs/get-multiple-users) so I just assumed that all existing users were active.